package.json

```diff
@@ -12,6 +12,7 @@
     "@babel/plugin-proposal-throw-expressions": "^7.12.1",
     "@babel/plugin-syntax-dynamic-import": "^7.0.0",
     "@babel/plugin-syntax-import-meta": "^7.10.4",
+    "@babel/polyfill": "^7.12.1",
```

Comment:
Was this intentionally added?

Reply:
yes, just because we are including it in the pack. We can actually remove this requirement.

Reply:
Removed